setup.py

```diff
@@ -34,7 +34,7 @@ def runSetup():
     gcs = 'google-cloud-storage==1.6.0'
     gcs_oauth2_boto_plugin = 'gcs_oauth2_boto_plugin==1.14'
     apacheLibcloud = 'apache-libcloud==2.2.1'
-    cwltool = 'cwltool==1.0.20190906054215'
+    cwltool = 'cwltool>=1.0.20190906054215'
```

Comment:
Thank you for these reviews and for adding me to the org.

I was focusing on the conditionals release. For this I needed to be able to use the `1.2-dev0` version of the specification. I implemented this in `cwltool` in a feature branch. In order to for Toil to accept this feature build, I unpinned the version.

Since we are aiming for the CWL 1.1 release we can have a **less than** dependency to some published `cwltool`.

In order to get it to work with the latest `cwltool` release (`2.0.20200126090152`) I had to move the schemaSalad dependency too. 

I wonder if we can actually remove the schemaSalad line in setup and allow cwltool to pull that in?